frontend/src/app/account/login/login.component.html

```diff
@@ -1,25 +1,47 @@
-<form [formGroup]="form" (ngSubmit)="login()">
+<img *ngIf="!justLoggedOut; else justLoggedOutArt" src="assets/images/login.png" class="art">
+<ng-template #justLoggedOutArt>
+  <img src="assets/images/logout.png" class="art">
+</ng-template>
+
+<div class="title" *ngIf="!justLoggedOut; else justLoggedOutTitle">
+  Log in to Arisia 2021!
+</div>
+<ng-template #justLoggedOutTitle>
+  <div class="title">
+    Thank you for attending Arisia 2021!
+  </div>
+</ng-template>
+<div class="content">
+  <form [formGroup]="form" (ngSubmit)="login()">
     <div class="form-group">
-        <label for="username">Username</label>
-        <input type="text" formControlName="username" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.username.errors }" />
-        <div *ngIf="submitted && f.username.errors" class="invalid-feedback">
-            <div *ngIf="f.username.errors.required">Username is required</div>
-        </div>
+      <label for="username">Username</label>
+      <input type="text" formControlName="username" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.username.errors }" />
+      <div *ngIf="submitted && f.username.errors" class="invalid-feedback">
+        <div *ngIf="f.username.errors.required">Username is required</div>
+      </div>
     </div>
     <div class="form-group">
-        <label for="password">Password</label>
-        <input type="password" formControlName="password" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.password.errors }" />
-        <div *ngIf="submitted && f.password.errors" class="invalid-feedback">
-            <div *ngIf="f.password.errors.required">Password is required</div>
-        </div>
+      <label for="password">Password</label>
+      <input type="password" formControlName="password" class="form-control" [ngClass]="{ 'is-invalid': submitted && f.password.errors }" />
+      <div *ngIf="submitted && f.password.errors" class="invalid-feedback">
+        <div *ngIf="f.password.errors.required">Password is required</div>
+      </div>
     </div>
     <div class="form-group">
-        <a routerLink="../forgot" class="btn btn-link">I forgot my username and/or password.</a>
+      <a routerLink="../forgot" class="forgot">I forgot my username and/or password.</a>
```

Comment:
i think this probably has to go away or like, do something other than what it does. also i'm not quite sure what lol. that's more of a todo than a comment on this.